ejector/ejector_config.go

```diff
@@ -131,6 +164,12 @@ func DefaultEjectorConfig() *EjectorConfig {
 		DataApiTimeout:                       60 * time.Second,
 		EthRpcRetryCount:                     3,
 		EthBlockConfirmations:                0,
+		ReferenceBlockNumberOffset:           10,
```

Comment:
64 blocks may make more sense, because that's how long it takes for eth finality